views/home/editreference.ejs

```diff
@@ -101,6 +101,14 @@
             </div>
 
             <div class="form-group">
+              <label for="referenceUrl" class="col-sm-4 control-label">Private Notes</label>
```

Comment:
Because we are calling this Private Notes, we should rename the existing notes "Public Notes", I think.